friends/src/store/reducers/index.js

```diff
@@ -0,0 +1,47 @@
+import {
+  FETCHING_FRIENDS,
+  FRIENDS_FETCHED,
+  FRIENDS_SAVED,
+  SAVING_FRIENDS,
+  UPDATING_FRIEND,
+  FRIEND_UPDATED,
+  DELETING_FRIEND,
+  FRIEND_DELETED,
+  ERROR
+} from '../actions';
+
+const initialState = {
+  friends: [],
+  fetchingFriends: false,
+  friendsFetched: false,
+  friendsSaved: false,
+  savingFriends: false,
+  updatingFriend: false,
+  friendUpdated: false,
+  deletingFriend: false,
+  friendDeleted: false,
+  error: null
+};
+
+export const friendReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCHING_FRIENDS:
+      return Object.assign({}, state, { fetching: true });
```

Comment:
I'm not sure this is working properly. The loading conditional never seems to trigger as true.